content/courses/advanced-ada/chapters/types.rst

```diff
@@ -1239,15 +1239,240 @@ Valid
 Unchecked Union
 ---------------
 
-.. admonition:: Relevant topics
+We've introduced variant records back in the
+:doc:`Introduction to Ada course <courses/intro-to-ada/chapters/more_about_types>`.
+In simple terms, a variant record |mdash| or *discriminated record* in Ada
+terminology |mdash| is a record with discriminants that allows for changing its
```

Comment:
But a discriminated record doesn't necessarily have a variant part, and often doesn't, so I would argue that they are not the same thing even if that is one of the primary motivations.

Reply:
Actually, this first phrase was taken from the "Ada for the Embedded C Dev." course. I was curious how this came to be and I found this comment:

https://github.com/AdaCore/learn/pull/404#discussion_r498873685

The reason I'm mentioning this here is because a correction is (most probably) needed in that course as well. Should I just remove the part about "record with discriminants" in both locations?

Reply:
No, I'm being too picky again. :-)